Magical-Kitties/Magical-Kitties-Save-the-Day.css

```diff
@@ -20,46 +16,73 @@ img.logo{
 }
 
 .score label {
-    text-align: center;
+  text-align: center;
 }
 
+.masthead {
+  text-transform: uppercase;
+  grid-column: 6 / 13;
+  line-height: 1.5rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  white-space: nowrap;
+}
 
-.attributes{
-    flex: 1 1 50%;
-    display: flex;
-    padding: 0;
-    text-align: center;
+.magical-kitties {
+  font-size: 32pt;
+  text-shadow: 1px 1px teal, 2px 2px teal, 3px 3px teal, 4px 4px darkgrey;
+  font-family: "Candal", sans-serif;
+  color: paleturquoise;
 }
 
+.save-the-day {
+  color: ghostwhite;
+  font-size: 16pt;
+  transform: translate(130px, 13px);
+  font-family: "Contrail One", sans-serif;
+  color: midnightblue;
+  text-shadow: 1px 1px ghostwhite, -1px -1px ghostwhite, -1px 1px ghostwhite, 1px -1px ghostwhite;
+}
 
-.attributes .score {
-    flex: 1 1;
-    display: flex;
-    justify-content: center;
-    flex-direction: column;
-    margin: 5px 0 5px 5px;
-    border: 8px solid gray;
-    border-radius: 40px;
-    min-height: 8.3em;
-    min-width: 8em;
+.kitty-token {
+  grid-column: 5 / 6;
 }
 
+.kitty-attributes {
```

Comment:
"attributes" was overriding the settings in the default character sheet tabs, so changed the name to avoid that